### ISSUE TITLE
Crash at initialisation when usb_midi is disabled in boot.py

### ISSUE THREAD

**Issue by x-magic** [adafruit/Adafruit_CircuitPython_MacroPad#33]
I know this is not a common use case but certainly can be addressed somehow. 

When I disabled `usb_midi` in `boot.py`: 
```python
import usb_midi
usb_midi.disable()  # Midi interface
```
Then initialise `adafruit_macropad` in `code.py`: 
```python
from adafruit_macropad import MacroPad
macropad = MacroPad()
```
It throws an `IndexError`: 
```
Traceback (most recent call last):
  File "code.py", line 13, in <module>
  File "adafruit_macropad.py", line 172, in __init__
IndexError: tuple index out of range
```
Which references https://github.com/adafruit/Adafruit_CircuitPython_MacroPad/blob/97e3712813ef35ce7c4b9d909c976cb4e7ebf41f/adafruit_macropad.py#L170-L177
Well this makes sense. As macropad lib also initialise hid devices and similar, I haven't tried if I disable HID would it crashes the library, too. 

And of course if I simply get rid of `adafruit_midi` library, macropad lib crashes as well. 

Maybe those things should really be modular? Thanks

**Comment by fivesixzero** [comment]
Since this is similar to #35 I did some testing to see if this could be addressed in a similar way. It looked like a `len` check on `usb_midi.ports` might be enough to see if its enabled or not.

I tested this out as a fix and it worked out well, so I've submitted the patch in PR (#37). I just noticed this got assigned to you, @makermelissa, so I hope I didn't step on any toes here by just tackling this real quick.

**Comment by ladyada** [comment]
thanks! we'll look to review :)

**Comment by fivesixzero** [comment]
Looks like this is all wrapped up with #37. Thanks to @dhalbert for the review and suggesting a slightly different, more effective approach. :)

**Comment by dhalbert** [comment]
Fixed by #37.

**Comment by x-magic** [comment]
Thank you everyone!
